Fix build with grep which brokenly prefers binary
Patch by Martin Vaeth <martin at mvath.de> from Feb 4, 2016
--- 1/make-rules/inputenc/Makefile.am
+++ 1/make-rules/inputenc/Makefile.am
@@ -43,7 +43,7 @@
 latin.xdy : latin1.xdy latin2.xdy latin3.xdy
 	echo ';; Generated from latin?.xdy' >latin.xdy
 	echo >>latin.xdy
-	sort -u latin?.xdy | grep -v '^;' >>latin.xdy
+	sort -u latin?.xdy | grep -va '^;' >>latin.xdy
 
 %.xdy %.idx :
 	$(PERL) $(srcdir)/make-testidx.pl TS1,T2A,T1 $* >$*.tex
